plugin/pkg/scheduler/factory/factory.go

```diff
@@ -338,6 +338,12 @@ func (f *ConfigFactory) CreateFromConfig(policy schedulerapi.Policy) (*scheduler
 			}
 		}
 	}
+	// issue: https://github.com/kubernetes/kubernetes/issues/43845
```

Comment:
I would rewrite the whole comment to something like:
`Providing HardPodAffinitySymmetricWeight in the policy config is new and preferred way of providing the value. Give it higher precedence when it is provided.`